yellowbrick/features/projection.py

```diff
@@ -234,9 +256,11 @@ def draw(self, Xp, y=None):
         """
         scatter_kwargs = self._determine_scatter_kwargs(y)
 
+        self.layout()
+
         if self.projection == 2:
             # Adds colorbar axis for continuous target type.
-            self.layout()
+            #            self.layout()
```

Comment:
Would prefer if you just removed this line of code rather than commented it out.

Reply:
Hehe, my bad! I accidentally skipped this!